endpoint/templates/name.model.spec(mongooseModels).js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var app = require('../../app');
+var <%= classedName %> = require('./<%= name %>.model');
+var <%= name %>;
```

Comment:
These `<%= name %>` should be `<%= cameledName %>` to avoid conflict with snake cased endpoint names.